roles/ansible/tower/config-ansible-tower-license/tasks/license.yml

```diff
@@ -1,17 +1,5 @@
 ---
 
-- block:
-  - name: Get Route Info
-    shell: "oc get route -n ansible-tower {{ kubernetes_deployment_name }}-web-svc -o json"
```

Comment:
oh, one more ... the `kubernetes_deployment_name` is defined in the `defaults/main.yml` file. Can you please clean-up that as well?